### DISCUSSION TITLE
Accented Letters Issue in "SumatraPDF-settings.txt" Shortcuts

### DISCUSSION THREAD

**Discussion by TonnoTotonno** [sumatrapdfreader/sumatrapdf#4051] [Q&A]
Hello everyone,

I'm wondering if there's a way to use accented letters in shortcuts. I've added the following lines to my ``SumatraPDF-settings.txt`` file:
```
Shortcuts [

	[
		Cmd = CmdScrollLeft
		Key = j
	]
	[
		Cmd = CmdScrollDown
		Key = k
	]
	[
		Cmd = CmdScrollUp
		Key = l
	]
	[
		Cmd = CmdScrollRight
		Key = ò
	]
]
```
However, the last remapping doesn't seem to work (and neither do other shortcuts with accented letters). Any suggestions?

**Comment by kjk** [comment]
Does your keyboard have a physical key for `ò` or is it a composite character (e.g. `Alt + o` or something like that)?

If composite, try the keys you're pressing (i.e. `Alt + o` or whatever it is that you press).

Also see if there's something in the log (`Ctrl + K` for command palette, `Show log`) related to that.

**Reply by TonnoTotonno** [reply]
My keyboard does have a physical key for ``ò``. It's a standard layout Italian keyboard, so all the accented characters are readily accessible.

However, the issue persists even when I try to remap other accented keys like ``ù``.

As for the log file, I've checked, but there doesn't seem to be anything relevant that could help troubleshoot the issue...

**Reply by kjk** [reply]
Sadly I don't have such keyboard to test so unless someone with such problem can debug the code and figure out why this happens, it won't get fixed.
